cleanlab/noise_generation.py

```diff
@@ -76,8 +76,8 @@ def noise_matrix_is_valid(noise_matrix, py, verbose=False):
 
 
 def generate_noisy_labels(y, noise_matrix, verbose=False):
-    """Generates noisy labels s (shape (N, 1)) from perfect labels y,
-    'exactly' yielding the provided noise_matrix between s and y.
+    """Generates noisy labels labels (shape (N, 1)) from perfect labels y,
```

Comment:
Can we style these differently? e.g. Generates noisy labels `labels` (shape (N, 1))